radio/src/lua/interface.cpp

```diff
@@ -64,19 +97,38 @@ int custom_lua_atpanic(lua_State * L)
 
 void luaHook(lua_State * L, lua_Debug *ar)
 {
-  instructionsPercent++;
-  if (instructionsPercent > 100) {
-    // From now on, as soon as a line is executed, error
-    // keep erroring until you're script reaches the top
-    lua_sethook(L, luaHook, LUA_MASKLINE, 0);
-    luaL_error(L, "CPU limit");
+  if (ar->event == LUA_HOOKCOUNT) {
+    instructionsPercent++;
+    if (instructionsPercent > 100) {
+      // From now on, as soon as a line is executed, error
+      // keep erroring until you're script reaches the top
+      // lua_sethook(L, luaHook, LUA_MASKLINE, 0);
```

Comment:
why did you comment out this line?

Reply:
I forgot to enable it back(I commented it out for testing). I will enable it back and test...